### PR TITLE
Fix orphaned resources validation.

### LINKED ISSUE #38: Validation of orphaned resources outputs wrong path
When validating for orphaned resources, errors are shown as '../../../' or something similar. This is because of an error in the sequence of parameters for calculating the relative path. Need to be fixed.

### PR DESCRIPTION
**Pull request by mtirionMSFT**
* Changed output of path of orphaned resources to be readable (instead "../../.."). Fixes #38

### CODE REVIEW

File: src/DocLinkChecker/DocLinkChecker/Helpers/MarkdownHelper.cs [hunk 1]
```diff
@@ -220,17 +221,29 @@ private static (PipeTable table, List<MarkdownError> errors)
...
-            int start = table.Span.Start;
-            while (start > 0 && markdown.Substring(start, 1) != "\n")
+            try
             {
-                start--;
-            }
+                pipeTable = new (markdownFilePath, table.Line, table.Column);
+
+                start = Math.Max(table.Span.Start, 0);
+                while (start > 0 && markdown.Substring(start, 1) != "\n")
```
Comment (Ellerbach) [inline src/DocLinkChecker/DocLinkChecker/Helpers/MarkdownHelper.cs:235]: thinking load: can you just remove replace the `\r\n` by `\n` (or `\r`) and simplify all this?

**Review by Ellerbach — COMMENTED** [review]
a remark maybe to simplify a bit the code but it can slow down the process

File: src/DocLinkChecker/DocLinkChecker/Helpers/MarkdownHelper.cs (hunk 1 above)
Comment (mtirionMSFT) [reply src/DocLinkChecker/DocLinkChecker/Helpers/MarkdownHelper.cs:235]: Could be an optimization. Will leave that for now to minimize the testing that needs to take place again.

**Review by Ellerbach — APPROVED** [review]

File: src/DocLinkChecker/DocLinkChecker/Helpers/MarkdownHelper.cs (hunk 1 above)
Comment (Ellerbach) [reply src/DocLinkChecker/DocLinkChecker/Helpers/MarkdownHelper.cs:235]: Fair enough. Approved.

**Pull request merged**